manubot/pandoc/util.py

```diff
@@ -30,21 +30,15 @@ def get_pandoc_info() -> Dict[str, Any]:
     }
     ```
     """
+    
     stats = dict()
-    for command in "pandoc", "pandoc-citeproc":
-        path = shutil.which(command)
-        stats[command] = bool(path)
-        if not path:
-            continue
-        version = subprocess.check_output(args=[command, "--version"], encoding="utf-8")
-        logging.debug(version)
-        version, *_discard = version.splitlines()
-        _discard, version = version.strip().split()
-        from packaging.version import parse as parse_version
-
-        version = parse_version(version).release
-        stats[f"{command} version"] = version
-        stats[f"{command} path"] = path
+    update_command_info("pandoc", stats)
+
+    if stats["pandoc version"] < (2, 11):
+        update_command_info("pandoc-citeproc", stats)
+    else:
+        stats["pandoc-citeproc"] = False
+
```

Comment:
If change function below according to suggestion, this could be updated like:

```suggestion
    stats = get_command_info("pandoc")

    if stats["pandoc version"] < (2, 11):
        stats.update(get_command_info("pandoc-citeproc"))
    else:
        stats["pandoc-citeproc"] = False
```